src/content/zendesk.js

```diff
@@ -20,7 +20,15 @@ setTimeout(() => {
       const getDescription = () => {
         const ticketId = document.querySelector('header div[data-selected=true]').attributes['data-entity-id'].value || ''
 
-        const input = elem.querySelector('input[aria-label=Subject]');
+        const language = document.querySelector("html").lang || 'en'
+
+        const translations = {
+          'en': 'Subject',
+          'de': "Betreff"
+        }
+
+        const elementName = translations[language] || 'Subject'
+        const input = elem.querySelector('input[aria-label='+elementName+']');
```

Comment:
As I said in the PR's review comment, this should be a better _locale independent_ selector.
It uses the `data-test-id` attribute that usually it's pretty stable, since it generally is used internally for E2E tests. 

```suggestion
        const input = elem.querySelector('input[data-test-id="omni-header-subject"]')
```